subsys/bluetooth/controller/ll_sw/nordic/lll/lll_master.c

```diff
@@ -85,7 +85,7 @@ static int prepare_cb(struct lll_prepare_param *prepare_param)
 	struct evt_hdr *evt;
 	u16_t event_counter;
 	u32_t remainder_us;
-	u8_t data_chan_use;
+	u8_t data_chan_use = 0;
```

Comment:
Have the assignment close to and just before the LL_ASSERT. (I assume then an initialization instructions can be avoided).